src/components/AdminTextPreview.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { applyScript } from '../lib/scripts'
+// I might have to add an ingestMethod just go get the first line of data.
+
+const AdminTextPreview = ({campaignData}) => {
+    console.log('LOOK HERE FIRST! ----------------------------- ADMINTEXTPREVIEW COMPONENT');
```

Comment:
I simplified the amount of arguments needed, to help with passing a full object that I could dissect in the ways that would be needed to pass a contact, a texter, a script, and the customFields. below.